packages/block-library/src/navigation-link/index.php

```diff
@@ -171,10 +194,10 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 
 	$html .= '</span>';
 
-	$html .= '</a>';
+	$html .= '</' . $item_tag . '>';
 	// End anchor tag content.
 
-	if ( $block->context['showSubmenuIcon'] && $has_submenu ) {
+	if ( isset( $block->context['showSubmenuIcon'] ) && $block->context['showSubmenuIcon'] && $has_submenu ) {
```

Comment:
This change seems unrelated. Is it needed?

Reply:
It was a suggestion from @adamziel to make sure we don't get PHP Notices sometimes.